test/e2e/helpers_test.go

```diff
@@ -291,3 +292,45 @@ func killKong(ctx context.Context, t *testing.T, env environments.Environment, p
 	}, kongComponentWait, time.Second)
 	t.Logf("kong container has %v restart after kill", after)
 }
+
+// buildImageLoadAddons creates addons to load KIC and kong images.
+func buildImageLoadAddonss(t *testing.T, images ...string) []clusters.Addon {
```

Comment:
```suggestion
func buildImageLoadAddons(t *testing.T, images ...string) []clusters.Addon {
```

+ the other changes that rely on this

Reply:
fixed.